.github/ISSUE_TEMPLATES/3-request-to-help-implement.yml

```diff
@@ -0,0 +1,25 @@
+name: Request to Help Implement
+description: Request help setting up this action in a repository/project that you help maintain.
+labels: ['[Type] Help Request']
```

Comment:
this doesn't match the current `help wanted` label, so either update here or update the repo label